libpod/inspect_data.go

```diff
@@ -0,0 +1,204 @@
+package libpod
+
+import (
+	"time"
+
+	digest "github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/specs-go/v1"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/projectatomic/libpod/libpod/driver"
+)
+
+// ContainerData handles the data used when inspecting a container
+type ContainerData struct {
+	Id              string           `json:"Id"`
+	Created         time.Time        `json:"Created"`
+	Path            string           `json:"Path"`
+	Args            []string         `json:"Args"`
+	State           *CtrState        `json:"State"`
+	ImageID         string           `json:"Image"`
+	ImageName       string           `json:"ImageName"`
+	ResolvConfPath  string           `json:"ResolvConfPath"`
+	HostnamePath    string           `json:"HostnamePath"` //not implemented
+	HostsPath       string           `json:"HostsPath"`    //not implemented
+	StaticDir       string           `json:"StaticDir"`
+	LogPath         string           `json:"LogPath"`
+	Name            string           `json:"Name"`
+	RestartCount    int32            `json:"RestartCount"` //not implemented
+	Driver          string           `json:"Driver"`
+	MountLabel      string           `json:"MountLabel"`
+	ProcessLabel    string           `json:"ProcessLabel"`
+	AppArmorProfile string           `json:"AppArmorProfile"`
+	ExecIDs         string           `json:"ExecIDs"` // don't know type yet, not implemented
+	HostConfig      *HostConfig      `json:"HostConfig"`
+	GraphDriver     *driver.Data     `json:"GraphDriver"`
+	SizeRw          int64            `json:"SizeRw,omitempty"`
+	SizeRootFs      int64            `json:"SizeRootFs,omitempty"`
+	Mounts          []specs.Mount    `json:"Mounts"`
+	Config          *CtrConfig       `json:"Config"`
+	NetworkSettings *NetworkSettings `json:"NetworkSettings"` // not implemented
+}
+
+// LogConfig holds the log information for a container
+type LogConfig struct {
+	Type   string            `json:"Type"`   // not implemented
+	Config map[string]string `json:"Config"` //idk type, not implemented
+}
+
+// RestarPolicy holds the container's policy on restart
```

Comment:
We aren't going to handle restart policy, so this can be removed

Reply:
I agree.

Reply:
removed